bundles/org.openhab.binding.fenecon/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -10,6 +10,7 @@
 		<label>FENECON Home</label>
 		<description>Thing for FENECON Home Device</description>
 		<category>Solarplant</category>
+		<semantic-equipment-tag>Inverter</semantic-equipment-tag>
```

Comment:
Are you sure this is simply an inverter? I see the word "inverter" only mentioned in the documentation as tag in the example, but I also see indications of a battery, and it's described as an "energy storage system".

Reply:
I am not sure.

Reply:
In that case it should probably be removed. At least I think having no tag is probably better than having a potentially wrong or misleading tag.

Reply:
@nixoso can you please comment?

Reply:
Consider perhaps the parent tag `PowerSupply` => https://github.com/openhab/openhab-core/blob/ad7612dba7963ef93c140bebb743aa3aab429928/bundles/org.openhab.core.semantics/model/SemanticTags.csv#L185

Reply:
@andrewfg I would also favour PowerSupply because it is in fact a mixture of inverter and battery.

Reply:
Done.